web/controllers/courseController/index.ts

```diff
@@ -2334,34 +2334,34 @@ async function groupMail(student: Student, courseId: number, subcourseId: number
 }
 
 /**
- * @api {POST} /course/:id/meeting/join JoinCourseMeeting
+ * @api {GET} /course/:id/subcourse/:subid/meeting GetCourseMeeting
  * @apiVersion 1.1.0
  * @apiDescription
- * Joins the BBB-Meeting for a given course
+ * Get the BBB-Meeting for a given subcourse
  *
- * This endpoint allows joining the BBB-Meeting of a course.
- * If the user is the instructor of the course the Meeting gets created with this call.
- * The other participants can only join after the instructor created the meeting with this endpoint
+ * This endpoint provides the BBB-Meeting of a subcourse.
  *
- * @apiName JoinCourseMeeting
+ * @apiParam (URL Parameter) {int} id ID of the main course
+ * @apiParam (URL Parameter) {int} subid ID of the subcourse
+ *
+ * @apiName GetCourseMeeting
  * @apiGroup Courses
  *
  * @apiUse Authentication
- * @apiUse Course
- *
- * @apiParam (JSON Body) {int} subcourseId ID of the subcourse of the course that should be joined
+ * @apiUse BBBMeeting
```

Comment:
I don't think that we will need the BBBMeeting in the API doc, because as far as I remember the BBBMeeting is only used internally and never exposed to the API?

Reply:
Yeah thats true.